regulations/static/regulations/css/less/typography.less

```diff
@@ -478,6 +478,36 @@ Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliqu
     line-height: 1.4;
 }
 
+.model-form-dash {
+    position: relative;
+    overflow: hidden;
+    display: inline-block;
+    width: 100%;
+    
+    span {
+        display: inline-block;
+        vertical-align: baseline;
+        zoom: 1;
+        *display: inline;
+        *vertical-align: auto;
+        position: relative;
+        padding-right: 0.4em;
+
+        &:after {
+            content: '';
+            display: block;
+            width: 1000px;
+            position: absolute;
+            top: 1.2em;
+            border-top: 1px solid @black;
+        }
+
+        &:after { left: 100%; }
+    }
+}
```

Comment:
A few things jump out at me, but nothing that should hold this pull request up.

1.  The `width: 1000px;` probably won't place nice on smaller screens.
2. Instead of using the `*` before the property name, which I'm assuming is to apply styling to older versions of IE, we can use IE conditional classes.
3. `&:after { left: 100%; }` should be split onto multiple lines.

All of that said, we can merge this in and I can play with getting the styles just so.